vars/im.md

```diff
@@ -12,6 +12,13 @@ This part of the pipeline provide steps for instant messaging.
        * [Configuration options](#configuration-options)
   * [Generic Configuration support](#generic-configuration-support)
   * [Examples](#examples)
+* [`im.teams()`](#imteams)
+  * [Arguments](#arguments)
+    * [`webhookUrlOrCredentialId`](#webhookUrlOrCredentialId)
```

Comment:
must be lowercase otherwise link is not working / marked as invalid in intellij

Reply:
that's weird, my intellij is not complaining. anyway, i will change it to follow conventions. check ✔️